var/spack/repos/builtin/packages/py-gmsh-interop/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyGmshInterop(PythonPackage):
+    """Interoperability between Python and Gmsh"""
+
+    homepage = "https://documen.tician.de/gmsh_interop"
+    git = "https://github.com/inducer/gmsh_interop"
+
+    maintainers = ["cgcgcg"]
```

Comment:
```suggestion
    maintainers("cgcgcg")
```
This new directive appends instead of overwrites so I'm still listed as a maintainer